lib/watir/wait.rb

```diff
@@ -172,26 +172,28 @@ def wait_while(deprecated_timeout = nil, deprecated_message = nil, timeout: nil,
     #   browser.text_field(name: "new_user_first_name").wait_until_present
     #
     # @param [Integer] timeout seconds to wait before timing out
+    # @param [Float] interval seconds to wait before each try
```

Comment:
These are no longer parameters, they are keywords, so they don't get documented like this.

Reply:
Ok, How should I document these?

Reply:
you don't need to. :)

Reply:
Oh, ok then.
Just to know, why keywords should not be documented? Ruby style? Thanks.

Reply:
And just one more Q, should I remove `timeout` arg document line? (was there already)

Reply:
Ok, I'm wrong, it looks like it should be using param: 
https://www.rubydoc.info/gems/yard/file/docs/Tags.md#option
But, looks like we should be specifying the name of it before the type:
https://www.rubydoc.info/gems/yard/file/docs/GettingStarted.md

Reply:
timeout as an ordered parameter is still available, just deprecated